lib/ApiRequestor.php

```diff
@@ -110,6 +112,8 @@ public function handleApiError($rbody, $rcode, $rheaders, $resp)
                 throw new Error\Authentication($msg, $rcode, $rbody, $resp, $rheaders);
             case 402:
                 throw new Error\Card($msg, $param, $code, $rcode, $rbody, $resp, $rheaders);
+            case 403:
+                throw new Error\Permissions($msg, $rcode, $rbody, $resp, $rheaders);
```

Comment:
Definitely a nit, but it looks like many of these other errors are named in the singular form. Any particular reason to prefer `Permissions` over `Permission` here?